src/VectorTypes_Native.f90

```diff
@@ -6,31 +6,7 @@
 ! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
 ! can be found in LICENSE.txt in the head directory of this repository.        !
 !++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
-!> @brief Utility module for defining vector types.
-!>
-!> The types of vectors defined in this module include a real vector type,
-!> and PETSc vector type.
-!>
-!> The objects are initialized with a parameter list. For valid reference lists
-!> see @ref VectorTypes::VectorTypes_Declare_ValidParams
-!> "VectorTypes_Declare_ValidParams".
-!>
-!> @par Module Dependencies
-!>  - @ref IntrType "IntrType": @copybrief IntrType
-!>  - @ref ExceptionHandler "ExceptionHandler": @copybrief ExceptionHandler
-!>  - @ref Allocs "Allocs": @copybrief Allocs
-!>
-!> @par EXAMPLES
-!> @code
-!> PROGRAM ExampleVector
-!>   TYPE(RealVectorType) :: vector
-!>
-!>   CALL vector%init(36)
-!>   CALL vector%set(1,10._SRK)
-!>   value=vector%get(1)
-!>   CALL vector%clear()
-!> ENDPROGRAM ExampleVector
-!> @endcode
+!> @brief Futility-native implementations of VectorType
```

Comment:
The example code is in the main modules, since those are the ones that should be user facing anyways. Just didn't get around to updating the module deps after the dust settled. Fixing it now